### ISSUE TITLE
Which version of Open5GS?

### ISSUE THREAD

**Issue by nishant412** [ryantheelder/open5gs_kamailio#2]
Hi,

Great tutorial on your blog. What versions of Open5GS did you use? The original blog refers to v1.3.0, is that the one the tutorial is written for?

Thanks in advance

**Comment by ryantheelder** [comment]
ChatGPT said:
ChatGPT

Hi,

Thank you for reading the blog and for your kind words! Unfortunately, I don’t have access to the setup anymore, but at the time of writing, I was using the latest available version of Open5GS. I hope that helps.

Best regards,

**Comment by ryantheelder** [comment]
Hi,

Thank you for reading the blog and for your kind words! Unfortunately, I don’t have access to the setup anymore, but at the time of writing, I was using the latest available version of Open5GS. I hope that helps.

Best regards,

**Comment by nishant412** [comment]
@ryantheelder Thanks for the response. Unfortunate you don't have the setup available. 

Just to make sure, the blog lists the date as March 20 2023. Did you happen to use Open5GS latest version as on that date or some older date? I will just look at git commits for open5gs repo, and choose the correct tag according to the date. Will be great help.

Thanks again

**Comment by ryantheelder** [comment]
@nishant412 

Yes, I set up and tested everything a few days before the blog's publication date. Using the Open5GS version available at that time should align closely with what I used.
